### DISCUSSION TITLE
parse_html and custom components

### DISCUSSION THREAD

**Discussion by arechnitzer** [elimintz/justpy#286] [Q&A] [answered]
I'm trying to make some custom components and I'd like to get them to work with the parse_html function. I've tried added the @jp.parse_dict decorator, but without success. 

If I wanted to get your "MyAlert" component from the docs to work as a tag recognised by jp.parse_html, how would I need to adapt it?

**Comment by elimintz** [comment, marked as answer]
Please see https://github.com/elimintz/justpy/issues/287
